docs/datagatherers/eks.md

```diff
@@ -0,0 +1,46 @@
+# EKS Data Gatherer
+
+The EKS *data gatherer* fetches information about a cluster from the AWS
+Kubernetes Service API.
```

Comment:
```suggestion
Elastic Kubernetes Service API.
```

Reply:
Fixed in 1319d079eda23ae69d48ae1e8f25d4fd8292ece1